CGPs/cgp-0075/mainet.json

```diff
@@ -3,9 +3,9 @@
     "contract": "GoldToken",
     "function": "approve",
     "args": [
-      "0x46421b8EF42b67Ae4E24BE599bD78144e5e12088",
+      "0x79d4881D34B6159aBcAb35Ad5029a4DE66c28626",
```

Comment:
This is another proposal that should not be edited. Please see other comment regarding how to resubmit.

Reply:
Also description of the proposal should be inside the text pf the md file, not in the conversation of GitHub